shell/assets/styles/global/_gauges.scss

```diff
@@ -31,7 +31,7 @@
 
 .resource-gauges {
   display: grid;
-  grid-column-gap: 10px;
+  grid-column-gap: 15px;
```

Comment:
@mantis-toboggan-md I changed this to match the col gap of the hardware resources but in harvester you are using this class as well. Is it ok on your side?

Reply:
This looks fine in Harvester. It actually matches the other gauges on the page better like this :)

Reply:
I thought the same too @mantis-toboggan-md . Thanks for the feedback! 🙌 🙇